apistrap/utils.py

```diff
@@ -34,3 +36,33 @@ def snake_to_camel(value):
     """
     result = "".join(x.capitalize() or "_" for x in value.split("_"))
     return result[0].lower() + result[1:]
+
+
+def get_function_perspective_globals(function, level=0, max_level=None):
+    globs = {}
+    if level != max_level:
+        stack = inspect.stack()
+        for frame in stack:
+            if frame.filename == inspect.getsourcefile(function):
+                globs.update(frame.frame.f_globals)
+    return globs
+
+
+def resolve_fw_decl(function, annotation, globs=None, level=0, search_stack_depth=2):
```

Comment:
missing param/return typehints & brief doc

maybe rename to `_resolve_fw_decl`?